src/platforms/hosted/README.md

```diff
@@ -1,7 +1,16 @@
 # PC-Hosted BMP
-Compile in src with "make PROBE_HOST=hosted". This needs minimal external
-support.  "make PROBE_HOST=hosted HOSTED_BMP_ONLY=0" will compile support for FTDI,
-STLink, CMSIS-DAP and J-Link probes, but requires external libraries.
+Compile with:
+``` sh
```

Comment:
These all need going through and correcting slightly - there should not be a space between the last backtick and "sh".